app/scripts/modules/kubernetes/src/pipeline/stages/runJob/configureJob.html

```diff
@@ -0,0 +1,115 @@
+<form name="form" class="container-fluid form-horizontal" style="padding: 0;">
+  <div>
+    <v2-modal-wizard heading="Configure Run Job"
+                     dismiss="ctrl.cancel">
+      <v2-wizard-page label="Basic Settings" mark-complete-on-view="false">
+        <stage-config-field label="Account">
+          <account-select-field component="ctrl.stage"
+                                field="account"
+                                provider="'kubernetes'"
+                                accounts="ctrl.accounts"
+                                on-change="ctrl.updateNamespaces()">
```

Comment:
I don't see this function defined anywhere - am I missing something?

Reply:
you aren't. a relic of something i was trying. will remove.